Dockerfile

```diff
@@ -110,6 +110,18 @@ RUN mkdir -p /usr/local/share/package-licenses \
   && cp /opt/intel/crypto-api-toolkit/LICENSE.md /usr/local/share/package-licenses/crypto-api-toolkit.LICENSE \
   && ls -l /opt/intel/
 
+WORKDIR /
+
+RUN cp /home/istio-proxy/sgx/include/* /usr/local/include/
+
+RUN wget https://golang.org/dl/go1.20.3.linux-amd64.tar.gz \
+  && tar -C /usr/local -xzf go1.20.3.linux-amd64.tar.gz \
+  && export PATH=$PATH:/usr/local/go/bin \
+  && export GOPATH=$HOME/go \
+  && export PATH=$PATH:$GOPATH/bin \
+  && git clone https://github.com/istio-ecosystem/hsm-sds-server.git \
```

Comment:
Use 'copy' other than 'git clone', so code change in local can take effect ASAP.